### DISCUSSION TITLE
Is it possible to disable telemetry data in Free-RASP-... ?

### DISCUSSION THREAD

**Discussion by SymphonyAISummit** [talsec/Free-RASP-Community#16] [Ideas]
We are using Free-RASP-Flutter plugin to detect Frida hooking and found plugin collects device data (Screen lock, Biometrics, HW-Backed Keystore, OS versions)

Is it possible/anyway to disable telemetry data in Free-RASP-Flutter Plugin?

**Comment by SymphonyAISummit** [comment]
Hi,

Any update?

**Reply by syakymchuk** [reply]
Hi @SymphonyAISummit , I have spotted your request to disable data collection.  

It is true that freeRASP collects the diagnostical information from every App/SDK instance.  According to current rules App Store the freeRASP service and Talsec as a company needs to be indicated as Data Processor. 

Only the commercial version of RASP SDK allows clients to address data collection to self-managed services or totally disable this feature.  If you are interested to discuss these options please [contact ](https://www.talsec.app/contact) us here or slot a meeting with me directly [here](https://calendly.com/sergiy-yakymchuk-1/60min?month=2023-03). 

Currently, Talsec is preparing the [lightRASP ](https://github.com/orgs/talsec/discussions/8)plan where this feature will be available.

**Comment by msikyna** [comment]
Hello @SymphonyAISummit ,

currently it is not possible to disable the telemetry data. It is also used in one of the reports describing how many of your users have screen lock, biometrics turned on or devices supporting HW-backed keystore.

Kind regards,
Matúš Šikyňa
